on-the-web/index.html

```diff
@@ -12,13 +12,13 @@ <h1>Catch up with OME</h1>
         <hr class="invisible">
         <section class="social-media-feature-section">
             <div class="social-media-feature-section-outer">
-                <h3 class="social-media-feature-section-headline"><i class="fa fa-twitter" aria-hidden="true"></i> Follow Us on Twitter</h3>
+                <h3 class="social-media-feature-section-headline"><i class="fa fa-twitter" aria-hidden="true"></i> Follow Us on Bluesky</h3>
                 <div class="social-media-feature-section-inner">
                     <div class="social-media-feature-section-feature top-left">
-                        <a class="account-profile" href="https://twitter.com/openmicroscopy" target="_blank"><img class="account-profile-pic" src="{{ site.baseurl }}/img/social-media/twitter_ome.png" alt="@openmicroscopy twitter profile pic"></a>
+                        <a class="account-profile" href="https://bsky.app/profile/openmicroscopy.org" target="_blank"><img class="account-profile-pic" src="{{ site.baseurl }}/img/social-media/twitter_ome.png" alt="@openmicroscopy bluesky profile pic"></a>
```

Comment:
Remove twitter_ome.png

Reply:
This is just the name of the, there is no twitter logo